src/views/About/about.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../vars';
+
+p{
+  margin-left: 15px;
+}
+a{
+  color: #0b91fd;
```

Comment:
I think you might not need to set the link color explicitly (Material-UI should handle that), but in case you do need to override it here, can you use a variable from `./src/_vars.scss` instead of the hex code?

My reasoning for that is, it is difficult to look at a hex code and know "Oh, that's the primary cyan from the Gordon color palette," which makes it difficult to maintain styles that use hex codes frequently. Using variables makes styles much more maintainable and easier to read.